source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2346 @@
+mmctl Command Line Tool (Beta)
+==============================
+
+The mmctl tool is a remote CLI tool for Mattermost which is installed locally and uses the Mattermost API. Authentication
+is done with either login credentials or an authentication token.
+
+Being installed locally allows a System Admin to run CLI commands even in instances where there is no access to the
+server (e.g., via SSH). This tool is currently in beta and can be used alongside the Mattermost CLI tool.
+In the future, the Mattermost CLI tool will be deprecated.
+
+This feature was developed to a large extent by community contributions and we'd like to extend our gratitude to the contributors who have
+worked on this project. We are currently accepting PRs for Help Wanted issues
```

Comment:
```suggestion
worked on this project. We are currently accepting pull requests for Help Wanted issues
```

Reply:
This one is not fixed anymore.

Reply:
That's my bad - I thought I'd committed it and must have over-written the change. Thanks!